doc/python/text-and-annotations.md

```diff
@@ -782,6 +782,47 @@ fig.add_annotation(
 
 fig.show()
 ```
+### Specifying Source Lines or Figure Notes on the Bottom of a Figure
+
+Container coordinates are an easy and robust way to put text -- such as a source line or figure note -- at the bottom of a figure.  Only the title command supports container coordinates.  If you have access to the HTML of the webpage in which you are embedding the graphic, either the source line, title, or both could go there instead.
+
+```
+import plotly.express as px
+df_iris = px.data.iris()
+fig = px.scatter(df_iris, x="sepal_width", y="sepal_length", color="species",
+                 size='petal_length', hover_data=['petal_width'])
+
+#Use the title for the source line
+fig.update_layout(
+        title=dict(text="Note: a near zero container coordinate is the most robust way to position this on the bottom.  Only the 'title' supports container coordinates.",
+                yref="container",
+                # A small positive y coordinate avoids cutting off the descending strokes of letters like y, p, and q.
+                y=0.005,
+                # Paper coordinates let us align this at either edge of the plot region 
```

Comment:
Thanks for the effort on this one @rl-utility-man 
I like the idea. When I tried the example, some of the text is cut off
<img width="1117" alt="image" src="https://github.com/user-attachments/assets/84a0970d-ef9f-4373-9d83-f91b84eef078" />
Makes me wonder if it's just maybe too specific of an example for the docs. Will it be difficult to get it working if your text looks different or you're running it in a specific environment?

Reply:
Good point about the cut off text!  I reworked the example with short text that works well at a wide variety of screen resolutions and added a comment warning about over long text getting cut off.  I also rewrote to draw out the general points I am trying to make with this example.  Let me know what you think of the revised version.  
![image](https://github.com/user-attachments/assets/5e08b953-c0e9-4ba7-afc8-162045d5f033)